### PR TITLE
Add Gelma's typos from "q" to "r"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -43103,8 +43107,11 @@ queing->queueing
 queires->queries
 queiried->queried
 queisce->quiesce
+quereis->queries
+queres->queries, queers,
 queriable->queryable
 quering->querying
+queris->queries
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:43114]: Perhaps `querist`, although probably rare?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -43081,6 +43083,8 @@ quantaties->quantities
 quantaty->quantity
 quantifiy->quantify
 quantifiying->quantifying
+quantit->quantity
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:43086]: `quantic` too?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -43103,8 +43107,11 @@ queing->queueing
 queires->queries
 queiried->queried
 queisce->quiesce
+quereis->queries
+queres->queries, queers,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:43111]: `queues` too?

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -45762,7 +45874,12 @@ repostiories->repositories
 repostiory->repository
 repostories->repositories
 repostory->repository
-repote->report, remote,
+repote->report, remote, repute,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:45877]: Also `repose`? If so, apply to all verbal forms.

Comment (int-y1) [reply codespell_lib/data/dictionary.txt:45877]: I couldn't find this typo.

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:43111]: I couldn't find this typo.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:43114]: `querist` is too rare.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Comment by DimitriPapadopoulos** [comment]
Thank you for your continuing involvement @int-y1.

**Pull request merged**